lib/domain/repository/phonebook_contact_repository.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:fluffychat/domain/model/contact/contact.dart';
+import 'package:flutter/foundation.dart';
```

Comment:
I think you should create a typedef `VoidCallback` and no need to import the flutter library, it's depends on the flutter framework is kind of scary :D

Reply:
I'll remove it, currently  don't care about it